### ISSUE TITLE
Provide way to get errors to MPD/WS users

### ISSUE THREAD

**Issue by adamcik** [mopidy/mopidy#1011]
We currently don't have any mechanism for letting users know when something goes wrong. This should probably be fixed.

**Comment by kingosticks** [comment]
Yes yes yes! And if the mechanism has the potential to be also used to
notify users of 'interesting events' e.g. losing the Spotify play token.

**Comment by adamcik** [comment]
Since I closed #1006 this issue should probably also consider the `warnings` case that the issue covered.

**Comment by adamcik** [comment]
This is also relevant for the case where core gets data from multiple backends and one or more crash.
